components/buttongroup/selection.md

```diff
@@ -5,15 +5,15 @@ description: Selected Items in the ButtonGroup for Blazor.
 slug: buttongroup-selection
 tags: telerik,blazor,button,group,selection
 published: True
-position: 2
+position: 3
```

Comment:
Consider position numbers that allow easier insertion of new articles in the future (e.g. divisors or 5 or 10)